### ISSUE TITLE
Manual Page 210 Filtered Mains Inlet and Switch are Upside Down

### ISSUE THREAD

**Issue by ChaosBlades** [VoronDesign/Voron-2#368]
The way the Mains Inlet and Switch are depicted on page 210 of the manual would make the printer power on by flipping the switch down instead of up. Also The power cable would be installed upside down compared to what is most common in the industry.

While this issue is very minor it has the potential to cause confusion and that is definitely something you do not want when mains voltage is concerned.

**Comment by DunarDolorgiet** [comment]
#372 addresses the manual part

**Comment by Jack-Lenders** [comment]
You're right. I got an DNS_PROBE_FINISHED_NXDOMAIN on my regular network. Switched to my mobile backup network and all link are working fine.

**Comment by mammothscientist** [comment]
Not a big issue mine fires up with no problem and is exactly the same as the manual.
